recipes/hitac/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.0" %}
+
+package:
+  name: hitac
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: python
+source:
+  url: https://gitlab.com/dacs-hpi/hitac/uploads/ea79d6d24da7b72038199f2cccebf724/hitac
+  md5: 94cc863fa214582c8ac50fdcb9347412
+
+requirements:
+  host:
+    - python >=3
+    - biopython
+    - pandas
+    - scikit-learn >=0.21.3
+    - numpy
+  run:
+    - python >=3
+    - biopython
+    - pandas
+    - scikit-learn >=0.21.3
+    - numpy
+
+test:
+  commands:
+    - hitac -h 2>&1 | grep "HiTaC"
+
+about:
+  home: https://gitlab.com/dacs-hpi/hitac
+  license: 'free software license'
```

Comment:
This should be `GPL3`. Also add `license_file: LICENSE`

Reply:
Done! I updated to license: 'GNU General Public License v3 (GPLv3)'

Reply:
Added the license_file now. I forgot about it before